src/Util/TestUtil.js

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
-import { mount } from 'enzyme';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-15';
+
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
Please move the global `Enzyme` configuration to an own file, e.g. `globals.js`.

Reply:
👍